packages/react-components/src/components/Alert.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { render, fireEvent } from '../test-utils';
+import { Alert, AlertSize, AlertType } from './Alert';
+
+const customClass = 'my-css-class';
+const bannerContent = 'This example content';
+const mockedFunction = jest.fn();
```

Comment:
1. `mockedFunction` is used only in one test, so I don't see a reason to create the variable here
1. The same with `customClass`, it's used only in one test so please move declaration closer to test.
1. `bannerContent` and default render function could be extracted. This is a good practice in component testing. By extracting I mean, instead of calling `render(<Alert` everytime in tests you can create function `renderComponent` and pass default props like banner content.

Let me know what do you think.